common/src/main/java/com/hortonworks/registries/common/util/FileStorage.java

```diff
@@ -20,13 +20,13 @@
 import java.util.Map;
 
 /**
- * Interface abstracting the upload and download of files (like parser jars, split/join or custom processor jars or any other files) for IoTaS.
- * IoTaS will provide a default file system based implementation which can be swapped by another implementation using fileStorageConfiguration
- * property in the iotas.yaml
+ * Interface abstracting the upload and download of files (like parser jars, split/join or custom processor jars or any other files) for Streamline.
+ * Streamline will provide a default file system based implementation which can be swapped by another implementation using fileStorageConfiguration
```

Comment:
Can we remove reference to streamline

Reply:
Right. Can we remove any streamline related information like parser jar or split/join etc mentioned in this description? Lets not even say Registry as this is common module which can be used in any component.